code/modules/antagonists/brother/brother.dm

```diff
@@ -171,6 +169,34 @@
 	if (prob(10))
 		brothers_left += 1
 
+/datum/team/brother_team/add_member(datum/mind/new_member)
+	. = ..()
+	if (!new_member.has_antag_datum(/datum/antagonist/brother))
+		add_brother(new_member.current)
+
+/datum/team/brother_team/remove_member(datum/mind/member)
+	if (!(member in members))
+		return
+	. = ..()
+	member.remove_antag_datum(/datum/antagonist/brother)
+	if (!member.current)
```

Comment:
```suggestion
	if (isnull(member.current))
```

Reply:
the new company i work at has much lower standards about truthiness, I am slipping...